tests/pytest/core/test_middleware_login_required.py

```diff
@@ -21,34 +34,22 @@ def require_login(request):
 
 
 @pytest.mark.django_db
-def test_login_auth_required(rf):
-    @decorator_from_middleware(LoginRequired)
-    def test_view(request):
-        pass
-
-    mock = create_autospec(test_view)
-
+def test_login_auth_required(rf, mocked_view, decorated_view):
     blocked_path = "/some/blocked/path"
     request = rf.get(blocked_path)
     initialize_request(request)
     require_login(request)
 
-    response = test_view(request)
+    response = decorated_view(request)
```

Comment:
This was a bug; it wasn't actually calling the decorated view, hence the mock not getting called.